Open Judge System/Workers/OJS.Workers.Common/AppSettings/Settings.cs

```diff
@@ -1,11 +1,11 @@
-﻿namespace OJS.Workers.Common
+﻿namespace OJS.Workers.Common.AppSettings
 {
     using System;
     using System.Configuration;
 
     using log4net;
 
-    public static class Settings
+    public static partial class Settings
```

Comment:
Why is this class `partial`? I think keeping it `static` is a better idea, given that it's literally just settings class